app/models/obligationsandsubmissions/Submission.scala

```diff
@@ -34,10 +34,16 @@ sealed trait SubmissionType extends EnumEntry {
 object SubmissionType extends Enum[SubmissionType] with PlayJsonEnum[SubmissionType] {
   val values: IndexedSeq[SubmissionType] = findValues
 
-  case object UKTR extends SubmissionType {
+  case object UKTR_CREATE extends SubmissionType {
     override val fullName: String = "UK Tax Return"
```

Comment:
```suggestion
    override val fullName: String = "UK Tax Return Create"
```

Should this be like that, so that it's consistent with `ORN_CREATE`?

Reply:
The UCD team and Business Analysts (BAs) are responsible for deciding what text should appear.